packages/mdc-tab-indicator/index.ts

```diff
@@ -93,4 +70,9 @@ class MDCTabIndicator extends MDCComponent {
   }
 }
 
-export {MDCTabIndicator, MDCTabIndicatorFoundation, MDCSlidingTabIndicatorFoundation, MDCFadingTabIndicatorFoundation};
+export {MDCTabIndicator as default, MDCTabIndicator};
+export * from './adapter';
+export * from './constants';
```

Comment:
We haven't been exporting constants from index - probably because they're already exposed off of the foundation, usually?

Reply:
Woops, removed